### DISCUSSION TITLE
About porting OE original features to modern Minecraft versions...

### DISCUSSION THREAD

**Discussion by vico93** [SirSquidly/Oceanic-Expanse#33] [General]
Oh, hi!

Sometime ago i opened (by mistake, should had make here instead) #21 asking if porting the original features of this mod to modern Minecraft versions was a possibility, you initially said yes but after a while closed the issues as invalid (i supposed because a port isnt on the radar).

Meanwhile i joined [a modding server](https://discord.gg/T5WaaSDate) that aims to port old and Forge-only to modern versions of Minecraft (on Fabric/Quilt), and put this mod on the list of requests for port. Even the mod being marked as MIT on CurseForge, i came here to ask if they could go forward in doing a unofficial port of OE features to Fabric/Quilt 1.20.x if they are interested.

Thanks a lot for this mod, it's unique and i really liked to play with it back on 1.12.2!

**Comment by vico93** [comment]
bump

**Comment by SirSquidly** [comment]
Oh yeah, if you want to port it, you are free to do so!

Some more exact things like any alterations, changes, or monetary stuff, I'd prefer if you private messaged me to discuss, albeit you don't _need_ to. The mod is in a state of flux, as I'm always trying to improve and refine it. If you have ANY ideas every, please contact me, I'd love to hear them!

**Reply by Boffinbox** [reply]
> _...alterations, changes, or monetary stuff, I'd prefer if you private messaged me to discuss,.._

where would be the best place to message you? i cannot find a contact link. i would like to discuss a particular piece of code in private.

**Reply by SirSquidly** [reply]
> where would be the best place to message you? i cannot find a contact link. i would like to discuss a particular piece of code in private.

Oh yeah, of course! Easiest would be on Discord, my account name is `sir_squidly` there with the same profile picture,, although my Google, Curseforge, and most accounts are about same.
